src/coreclr/jit/instr.h

```diff
@@ -297,22 +291,33 @@ enum insOpts : unsigned
     INS_OPTS_H_TO_D,      // Half to Double
 
     INS_OPTS_S_TO_H,      // Single to Half
-    INS_OPTS_D_TO_H       // Double to Half
+    INS_OPTS_D_TO_H,      // Double to Half
 
 #if FEATURE_LOOP_ALIGN
-    , INS_OPTS_ALIGN      // Align instruction
+    INS_OPTS_ALIGN,       // Align instruction
 #endif
+
+    // There should be no overlap between non-SVE and SVE values,
+    // so reset value to 1 here
+    INS_OPTS_SCALABLE_B = 1,
```

Comment:
we don't need resetting anymore. also, you can pretty much revert changes done in `enum insOpts`